recipes/eleuther_eval.py

```diff
@@ -102,20 +102,20 @@ def _model_generate(self, *args, **kwargs):
 
 
 class EleutherEvalRecipe(EvalRecipeInterface):
-    """This recipe runs evaluation on a trained model using EleutherAI's eval harness.
-    This assumes the user has the EleutherAI eval harness installed.
+    """
+    This recipe runs evaluation on a trained model using EleutherAI's eval harness.
+    This assumes the user has the EleutherAI eval harness installed. See
+    https://github.com/EleutherAI/lm-evaluation-harness for more details.
 
-    This recipe supports:
-        - Single GPU evaluation
-        - Loading model in fp32 or bf16
+    Features:
+        - Single GPU evaluation. Multi-GPU evaluation is currently not supported.
+        - Loading model in fp32 or bf16. Fp16 is currently not supported.
         - Any task from the EleutherAI eval harness that is *not* free generation
 
-    Assumptions:
-        - Evaluation is launched with the Tune CLI (recommended)
-        - User has the EleutherAI eval harness installed, see https://github.com/EleutherAI/lm-evaluation-harness
+    We recommend launching evaluation using the tune CLI:
 
-    The following configs can be used to run this recipe:
-        - eleuther_eval.yaml
+        tune run eleuther_eval --config llama2_eleuther_eval \
+        tasks=["truthfulqa_mc2", "hellaswag"]
```

Comment:
I think the space is helpful visually?

Reply:
Don't think it adds anything, and it might be confusing if a user copies and launches this command and it fails because of the space. But this is very minor, so your call.

A separate point - why not just make these tasks default in the config?

Reply:
Did you test that it actually runs with the space? Personally I have had issues with this in the past

Reply:
Yes I've run the command with the space and it does not work

Reply:
Ah ok